static/app/stores/groupingStore.tsx

```diff
@@ -239,12 +275,53 @@ const storeConfig: Reflux.StoreDefinition & Internals & GroupingStoreInterface =
     );
 
     const responseProcessors: ResponseProcessors = {
-      merged: item => {
-        // Check for locked items
-        this.setStateForId(this.unmergeState, item.id, {
-          busy: item.state === 'locked',
+      merged: items => {
+        const newItemsMap: {[key: string]: Fingerprint} = {};
+        const newItems: Fingerprint[] = [];
+
+        items.forEach(item => {
+          if (!newItemsMap[item.id]) {
+            const newItem = {
+              id: item.id,
+              label: item.label,
+              parentId: item.parentId,
+              parentLabel: item.parentLabel,
+              eventCount: 0,
+
+              // Those properties are correct since the server returns items in
+              // descending order of lastSeen
+              lastSeen: item.lastSeen,
+              latestEvent: item.latestEvent,
+              children: [],
+            };
```

Comment:
```suggestion
            const newItem = {
              eventCount: 0,
              children: [],
              // lastSeen and latestEvent properties are correct
              // since the server returns items in
              // descending order of lastSeen
              ...item
            };
```